primeqa/tableqa/trainer/tableqa_trainer.py

```diff
@@ -0,0 +1,59 @@
+from transformers import Trainer
```

Comment:
minor, but for MRC this folder is called "trainers"

Reply:
renamed.